### DISCUSSION TITLE
error after install rsuite on next.js

### DISCUSSION THREAD

**Discussion by mahmudipur** [rsuite/rsuite#1769] [General]
hey guys!
I have recently install rsuite on my project using [this](https://rsuitejs.com/guide/use-next-app/) structure and now I have some issue after 
`npm run dev`
![7e39b37f-76a3-421b-a9d3-59a2f0e7f859](https://user-images.githubusercontent.com/57182874/124377028-15f25780-dcbf-11eb-89d4-5e3afb342fb9.jpg)
can anyone help me to solve this?

**Comment by akshay8033** [comment]
Facing the same issue.

**Comment by simonguo** [comment]
Similar issue reference: https://github.com/rsuite/rsuite/issues/1320#issuecomment-883038430
